testing/runner.ts

```diff
@@ -95,74 +93,110 @@ export async function getMatchingUrls(
 
   return matchingLocalUrls.concat(matchingRemoteUrls);
 }
+
+export interface RunTestModulesOptions extends RunOptions {
```

Comment:
Can you rename `RunOptions` to `RunTestsOptions`, it's the same name as options for `Deno.run`

Reply:
👍 That extends the scope a bit so I'll save it for the end.